app/tests/e2e/test_verifier.py

```diff
@@ -121,12 +122,18 @@ async def test_accept_proof_request(
     )
     proof_request_v1.connection_id = bob_and_alice_connection["alice_connection_id"]
     proof_dict["connection_id"] = bob_and_alice_connection["alice_connection_id"]
-    time.sleep(3)
+
     proof_req_res = await alice_member_client.post(
         BASE_PATH + "/send-request",
         json=proof_request_v1.dict(),
     )
 
+    assert check_webhook_state(
```

Comment:
Same here

Reply:
Been trying to fix accept proof request for quite a while last night. Suspect there is a bug in there. Will open another issue for this.